README.md

```diff
@@ -147,6 +147,91 @@ expect(cat.sound(), "Purr");
 //using real object
 expect(cat.lives, 9);   
 ```
+
+## Strong mode compliance
+
+Unfortunately, the use of the arg matchers in mock method calls (like `cat.eatFood(any)`)
+violates the [Strong mode] type system. Specifically, if the method signature of a mocked
+method has a parameter with a parameterized type (like `List<int>`), then passing `any` or
```

Comment:
I don't think so. This is why it's not such a prominent problem. Just a little prominent:

```
$ cat lib/dogs.dart
import 'mockito.dart';

class Dog {
  bool eatFood(String food) => true;
  bool eatFood2(List<String> foods) => true;
}

class MockDog extends Mock implements Dog {}

void fn() {
  var dog = new MockDog();
  when(dog.eatFood(any)).thenReturn(true);
  when(dog.eatFood2(any)).thenReturn(true);
}
$ dartanalyzer --strong lib/dogs.dart
Analyzing [lib/dogs.dart]...
[warning] Unsound implicit cast from dynamic to List<String> (/Users/srawlins/code/dart-mockito/lib/dogs.dart, line 13, col 21)
1 warning found.
```